request.js

```diff
@@ -3,6 +3,8 @@
 var tls = require('tls')
 var http = require('http')
 var https = require('https')
+var http2 = require('./lib/http2').default;
```

Comment:
Nit maybe, but I think there's a way to publish TypeScript modules still as commonjs. I think it's `export = ` last I checked. 

Mostly just the default export is kinda wack in a project which is mostly CommonJS.

Reply:
Yeah we still need to look into the pipeline of it all, mainly focus on feature completeness and readiness.
Will fix this up before making the PR ready for review